sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/Ecdsa.java

```diff
@@ -52,10 +52,11 @@ public boolean verify(byte[] digest, byte[] signature) throws GeneralSecurityExc
             return verify.verify(signature);
         }
 
-        private void checkDigestLength(byte[] digest) {
-            if (digest.length != getDigestLength()) {
-                throw new IllegalArgumentException("Invalid digest length.");
-            }
+    }
+
+    private void checkDigestLength(byte[] digest) {
```

Comment:
I've been staring at this for 30 seconds. It looks exactly the same to me. LOL.

Reply:
Its a false positive.
Flagged as a  private method not being used, but is being used by a class inside the class.

Reply:
The code inside the method is exactly the same. However, the change is the method is now outside the inner class. As this is a private method and none of the outer class code calls this private method, it results in a spotbugs error even though it's accessed from the inner class. It was one of the issues Vinay ran into before preview 2 release.

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=84027&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=89c92298-345c-5542-6ef5-ddc11b4f6d47&l=50

Reply:
Ahhhh I see!